openassessment/templates/openassessmentblock/response/oa_response_submitted.html

```diff
@@ -44,13 +44,7 @@ <h3 class="submission__answer__display__title">{% trans "Your Response" %}</h3>
 
                 {% include "openassessmentblock/oa_submission_answer.html" with answer=student_submission.answer answer_text_label="Your response to the question above:" %}
 
-                {% if allow_file_upload and file_url %}
-                <h3 class="submission__answer__display__title">{% trans "Your Image" %}</h3>
-
-                <div class="submission__answer__display__image">
-                    <img class="submission--image" alt="{% trans "The image associated with your submission." %}" src="{{ file_url }}"/>
-                </div>
-                {% endif %}
+                {% include "openassessmentblock/oa_uploaded_file.html" with file_upload_type=file_upload_type file_url=file_url  header="Your Uploaded File" class_prefix="submission__answer" %}
```

Comment:
@muzaffaryousaf I don't understand your comment. They need to be translated everywehre so they display properly